src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -66,32 +66,35 @@ class CollapsibleCourseRunFields extends React.Component {
         {fields.map((courseRun, index) => (
           <Collapsible
             title={formatCourseRunTitle(courseRuns[index])}
-            iconId={`collapsible-icon-${index}`}
+            key={`collapsible-run-${courseRun}`}
+            iconId={`collapsible-icon-${courseRun}`}
```

Comment:
I think the `key` and `iconId` need a unique string, but they'll just get `[object Object]` on the end right now

Reply:
courseRun comes as a string, string-thing-
<img width="742" alt="Screen Shot 2019-04-01 at 2 37 37 PM" src="https://user-images.githubusercontent.com/1559107/55351290-ec1cab80-548b-11e9-8270-618c140883c1.png">
course_runs[0]

Reply:
Ok, cool. The snapshots deceived me